flink-cdc-base/src/main/java/com/ververica/cdc/connectors/base/utils/ObjectUtils.java

```diff
@@ -26,13 +26,21 @@ public class ObjectUtils {
 
     /**
      * Returns a number {@code Object} whose value is {@code (number + augend)}, Note: This method
-     * does not consider number overflow because we don't want to change the object type.
+     * will return max value of number type,if number overflows.
      */
     public static Object plus(Object number, int augend) {
         if (number instanceof Integer) {
-            return (int) number + augend;
+            try {
+                return Math.addExact((Integer) number, augend);
```

Comment:
What about throwing the `ArithmeticException` directly if the result overflows, and catch the exception in `ChunkSplitter`? Returning Integer.MAX or Long.MAX on overflow is a bit confusing and ambiguous.

Reply:
Thks review,I will fix this.